README.md

```diff
@@ -71,9 +71,13 @@ To run the test script located [here](test/TemplateTest.ps1), Execute the follow
 - blobTrigger
 - timerTrigger
 
+## License
+
+This project is under the benevolent umbrella of the [.NET Foundation](http://www.dotnetfoundation.org/) and is licensed under [the MIT License](https://github.com/Azure/azure-webjobs-sdk-templates/blob/master/LICENSE.txt)
```

Comment:
Nit: Can use a relative link for license.txt. Would make the License link work on dev branch as well.